docs/docs/guides/tracking/objects.md

```diff
@@ -10,8 +10,9 @@ Weave's serialization layer saves and versions Python objects.
 ## Publishing an object
 
 ```python
+import weave
 # Initialize tracking to the project 'cat-project'
-weave.init('cat-project')
+weave.init('intro-example')
```

Comment:
Need to update comment in line above this

Reply:
👍

Reply:
Fixed